woocommerce-checkout-non-pci-gateway/view/js/checkout_frames.js

```diff
@@ -6,8 +6,8 @@ jQuery(document).ready(function() {
             Frames.init(window.CheckoutApiEmbConfig);
         }
 
-        jQuery(document).ready(function() {
-            Frames.init(window.CheckoutApiEmbConfig);
+        // jQuery(document).ready(function() {
```

Comment:
@avish-bisbeehurry-cko is there a reason this has been kept?

Reply:
No this should have been removed. I will update it.